### PR TITLE
Address registry

### PR DESCRIPTION
**Pull request by 0xjocke**

### CODE REVIEW

File: markets/bfp-market/contracts/modules/LiquidationModule.sol [hunk 1]
```diff
@@ -306,8 +313,14 @@ contract LiquidationModule is ILiquidationModule {
         if (position.size == 0) {
             revert ErrorUtil.PositionNotFound();
         }
-
-        uint256 oraclePrice = market.getOraclePrice(ORACLE_MANAGER);
+        ISynthetixSystem synthetix = ISynthetixSystem(SYNTHETIX_CORE);
+        uint256 oraclePrice = market.getOraclePrice(
+            AddressRegistry.Data({
+                synthetix: ISynthetixSystem(SYNTHETIX_CORE),
```
Comment (davidvuong) [inline markets/bfp-market/contracts/modules/LiquidationModule.sol:319]:
```suggestion
                synthetix: synthetix,
```

?

File: markets/bfp-market/contracts/storage/Position.sol [hunk 2]
```diff
@@ -597,10 +582,22 @@ library Position {
...
+            addresses
+        );
+
+        healthData.healthFactor = marginValues.discountedMarginUsd.divDecimal(mm);
 
         // Calc the price PnL.
         healthData.pnl = size.mulDecimal(price.toInt() - positionEntryPrice.toInt());
 
+        return healthData;
+    }
```
Comment (davidvuong) [inline markets/bfp-market/contracts/storage/Position.sol:600]: Invalid syntax?

**Review by davidvuong — CHANGES_REQUESTED** [review]
`Position.sol` is broken

File: markets/bfp-market/contracts/modules/LiquidationModule.sol (hunk 1 above)
Comment (0xjocke) [reply markets/bfp-market/contracts/modules/LiquidationModule.sol:319]: Yea mistake

**Comment by davidvuong** [comment]
> [synthetix-v3]: [@synthetixio/buyback-snx]: Error: could not download cannon package data from "QmfDgnQyUNCkKg9amftCFiE4AyPt933dbCkU3rAGWDJ5mV": AxiosError: Request failed with status code 502

image

**Review by davidvuong — APPROVED** [review]

**Pull request merged**